WolvenKit.App/Services/IUpdateService.cs

```diff
@@ -0,0 +1,10 @@
+using System.Threading.Tasks;
+
+namespace WolvenKit.App.Services;
+
+public interface IUpdateService
+{
+    public Task<bool> IsUpdateAvailable();
```

Comment:
We had similar functions in `AppViewModel` on wkit startup, is this what you're removing from there and reimplementing as a service here?